src/profile.js

```diff
@@ -31,11 +30,11 @@ const start = (msg) => {
 
     const profiler = logging.startTimer();
     return {
-        end: (endMsg) => {
+        end: (endMsg, level = "debug") => {
             const message = endMsg || msg;
             profiler.done({
                 message: `PROFILE(end): ${message}`,
-                level: "debug",
+                level,
```

Comment:
Does this `level` get overwritten inside `logging.js`? I'm wondering if this level could change depending on the `level` flag passed from the arguments.

Reply:
There are two levels. This `level` is the level of the message being logged. The `level` passed in from arguments is the `level` that gets actually logged. 

So, the second applies a filter to the first.  The level of a message doesn't change - only whether it is actually written to the output transport.